### ISSUE TITLE
Quixe error in React application

### ISSUE THREAD

**Issue by meteoratlas** [vorple/vorple#57]
Hi there! I'm trying to get vorple up and running in a React application (following this example: https://vorple-if.com/docs/js-library.html), but I get the following error in the console, and nothing renders: 

`quixe.js:63 Uncaught ReferenceError: Quixe is not defined
    at ./node_modules/vorple/dist/quixe/quixe.js (quixe.js:63:1)`

This can be reproduced by:
1. Create a new app using Create React App
2. In the new directory, run `npm install vorple`
3. Replace the contents of the default App.js file with the example App file on the above vorple docs page (Examples -> React project)

The example not using a framework seems to work just fine when I try that.

I'm not sure if this is a library error or if I'm simply missing something, but any help would be appreciated. Thanks!

**Comment by juhana** [comment]
It seems to have something to do with either Create React App being more strict with implicitly created global variables, or the npm package has been incorrectly bundled. In either case, I'll fix it – in the meanwhile you can get it running with this temporary workaround: 

Create a file called "QuixeFix.js" with the following content:

```js
window.Quixe = undefined;
window.GiDispa = undefined;
window.GiLoad = undefined;
window.holdvar = undefined;
```

Import the file in App.js *before* importing Vorple, for example:

```js
import "./QuixeFix";
import vorple from "vorple";
```

Another option is to install version 3 instead (`npm install vorple@3.2.8`) which doesn't have the bug. It's practically identical to the latest version (4.0.0) and the only difference is what version of Inform 7 the extensions work with. Both 3.2.8 and 4.0.0 versions of the Javascript library are compatible regardless of which version of the extensions are used.

**Comment by meteoratlas** [comment]
Good to know - I've installed version 3 and everything is working correctly now. Thank you very much for your help - I'll leave this issue open for now if you want to reference it re: the eventual solve for this, but I have no problem using the older version, so if you want to close it be my guest :)

**Comment by juhana** [comment]
This is now fixed in release 4.0.1.
